src/components/AssetsClearFiltersButton/AssetsClearFiltersButton.test.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { Button } from '@edx/paragon';
 
 import AssetsClearFiltersButton from './index';
-import courseDetails from '../../utils/testConstants';
+import { courseDetails } from '../../utils/testConstants';
```

Comment:
All the following courseDetails changes are because I made courseDetails a named export of testConstants instead of the default export, since I didn't think it made sense for it to be the default export.